front/src/Phaser/Game/GameScene.ts

```diff
@@ -980,6 +997,21 @@ ${escapedMessage}
         this.dirty = true;
     }
 
+    private getIndexForTileType(tileType: string | number): number | null {
+        if (typeof tileType == "number") {
+            return tileType;
+        }
+        for (const tileset of this.mapFile.tilesets) {
+            if (tileset.tiles) {
+                for (const tilesetTile of tileset.tiles) {
+                    if (tilesetTile.type == tileType) {
```

Comment:
We should avoid relying on the "tile type" (because it can potentially be used in Tiled itself to set properties automatically https://doc.mapeditor.org/en/stable/manual/custom-properties/#typed-tiles)

Instead, I think we should use a custom "name" property that we can put on a tile (and document that with a screenshot in the doc)

Also, :+1: for building a map of "name" => Tile (as suggested by @jonnytest1 .

Finally, this `getIndexForTileType` should be in `GameMap` and not in `GameScene`